models/omop/provider.sql

```diff
@@ -1,5 +1,5 @@
 SELECT
-    row_number() OVER (ORDER BY (SELECT null)) AS provider_id
+    row_number() OVER (ORDER BY provider_state, provider_city, provider_zip, provider_id) AS provider_id
```

Comment:
could we just use provider_id? it should be the PK from the source Synthea table right?

Reply:
yeah we could ~ I was going for something that would order somewhat logically (rather than UUIDv4s), but that's again not necessary

Reply:
ahh i see! just for readability/neatness. i'm fine to keep it this way then :)